.travis/Dockerfile.windows-server

```diff
@@ -0,0 +1,28 @@
+FROM mcr.microsoft.com/windows/servercore:1809
+ENV LANG C.utf8
+
+# Install MSYS2. See: https://www.msys2.org/docs/ci/
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+RUN [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; \
+    Invoke-WebRequest -UseBasicParsing -uri "https://github.com/msys2/msys2-installer/releases/download/nightly-x86_64/msys2-base-x86_64-latest.sfx.exe" -OutFile msys2.exe; \
+    .\msys2.exe -y -oC:\; \
```

Comment:
Is renaming the installer on download a good idea? Using the same name as the project and probably some other executable does not seem too wise. You do specify `./msys2.exe` so it's probably enough, though.